src/poetry/console/commands/about.py

```diff
@@ -10,9 +10,16 @@ class AboutCommand(Command):
     description = "Shows information about Poetry."
 
     def handle(self) -> None:
+        import poetry.core
+
+        from poetry.__version__ import __version__
+
         self.line(
-            """\
-<info>Poetry - Package Management for Python</info>
+            f"""\
+<info>Poetry - Package Management for Python
+
+Version: {__version__}
+Poetry-Core Version: {poetry.core.__version__}</info>
```

Comment:
We should not rely on `__version__` here, but should rather use importlib. We should consider dropping it in the future.

Reply:
Using `importlib` was my first try, but for whatever reason I changed it. Happy to use it again.

Reply:
Done.